simulator/util/dummy_perception_publisher/src/empty_objects_publisher.cpp

```diff
@@ -24,7 +24,7 @@ class EmptyObjectsPublisher : public rclcpp::Node
 public:
   EmptyObjectsPublisher() : Node("empty_objects_publisher")
   {
-    empty_objects_pub_ = this->create_publisher<autoware_perception_msgs::msg::DynamicObjectArray>(
+    empty_objects_pub_ = this->create_publisher<autoware_auto_perception_msgs::msg::TrackedObjects>(
```

Comment:
I think empty_object_publisher publishes predicted results.

Is it shoule be `PredictedObjects`?

Reply:
fix at 33f634f4077518eb5f77e24c08d39a98bc9d9655